pkg/networkmanager/interfaces.js

```diff
@@ -720,13 +720,25 @@ export function NetworkManagerModel() {
         case 11: return 'vlan';
         case 12: return 'adsl';
         case 13: return 'bridge';
-        case 14: return 'loopback';
+        case 14: return 'generic';
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18850-20230526-094431-7eb31071-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)